src/lib/Libifl/pbsD_modify_resv.c

```diff
@@ -67,33 +67,50 @@ char *
 pbs_modify_resv(int c, char *resv_id, struct attropl *attrib, char *extend)
 {
 	struct attropl *pal = NULL;
-	int		rc = 0;
-	char		*ret = NULL;
+	int rc = 0;
+	char *ret = NULL;
+	int i;
+	svr_conn_t **svr_conns = get_conn_svr_instances(c);
```

Comment:
you need to also handle the case where the IFL is invoked with the instance fd. In that case get_conn_svr_instances(c) will return NULL.

Reply:
Thanks Nithin. I have handled this as well. Please look into my latest commit.